lib/wallet.js

```diff
@@ -28,8 +29,9 @@ function computeSeed (masterSeed) {
 }
 
 function fetchWallet (settings, cryptoCode) {
-  return fs.readFile(options.seedPath, 'utf8')
-    .then(hex => {
+  return fs.readFile(options.mnemonicPath, 'utf8')
+    .then(mnemonic => {
+      const hex = bip39.mnemonicToEntropy(mnemonic.split('\n').join(' ').trim())
```

Comment:
This should be done in a separate module so that it's consistent in different places in the code. Should return a buffer.